devicesimulation/armtemplates/DeviceSimulationNoIoTHub.json

```diff
@@ -0,0 +1,535 @@
+{
```

Comment:
since current remote monitoring solution is using basic and standard naming. How about we follow this convention to use 'basic-no-iothub.json' and 'basic.json'?

Reply:
we also need parameter file here.

Reply:
IoTSuite do not need the parameter file. It will generate the parameters

Reply:
pcs cli will use it.

Reply:
It is out of scope of this PR. Just like we did not did not include the "all-in-one" yml file